.github/workflows/lint-and-test.yml

```
@@ -9,9 +9,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v4.1.1
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
       - name: Setup node
-        uses: actions/setup-node@v4
+        uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8 # v4
       - name: Set up Ruby
```

```suggestion
      - name: Setup Ruby, JRuby, and TruffleRuby
```

Refs: https://github.com/OpenINF/.github/commit/76b91dfb87abcddbbf4bb925f3610bc07c5aaf6b